File: Semestrul 4/MPP2/Problema8_Csharp_ORM/Networking/dto/AgeGroupDTO.cs
namespace Networking.dto;

[Serializable]
public class AgeGroupDTO
{
    public string Name { get; set; }
    public int MinAge { get; set; }
    public int MaxAge { get; set; }

    public AgeGroupDTO(string name, int minAge, int maxAge)
    {
        Name = name;
        MinAge = minAge;
        MaxAge = maxAge;
    }

    public override string ToString()
    {
        return $"AgeGroupDTO[{Name} {MinAge} {MaxAge}]";
    }
}

File: Semestrul 4/MPP2/Problema8SC/Problema8SC/Problema8Networking/src/main/java/problema8/network/dto/AgeGroupDTO.java
package problema8.network.dto;

import java.io.Serializable;

public class AgeGroupDTO implements Serializable {
    private String name;
    private int minAge;
    private int maxAge;

    public AgeGroupDTO(String name, int minAge, int maxAge) {
        this.name = name;
        this.minAge = minAge;
        this.maxAge = maxAge;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getMinAge() {
        return minAge;
    }

    public void setMinAge(int minAge) {
        this.minAge = minAge;
    }

    public int getMaxAge() {
        return maxAge;
    }

    public void setMaxAge(int maxAge) {
        this.maxAge = maxAge;
    }

    @Override
    public String toString() {
        return "AgeGroupDTO[" + name + " " + minAge + " " + maxAge + "]";
    }
}
